google/cloud/bigquery/v2/minimal/benchmarks/latency_readonly_project_apis.cc

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/bigquery/v2/minimal/benchmarks/benchmarks_config.h"
+#include "google/cloud/internal/absl_str_cat_quiet.h"
+#include "google/cloud/internal/build_info.h"
+#include "google/cloud/internal/compiler_info.h"
+#include "google/cloud/internal/getenv.h"
+#include "absl/strings/match.h"
+#include <functional>
+#include <sstream>
```

Comment:
Please cleanup includes.  Or simply do not add these files until they are needed.